python/src/numerous/frameworks/dash.py

```diff
@@ -0,0 +1,6 @@
+from numerous.frameworks.flask import FlaskSession
+
+class DashSession(FlaskSession):
+    pass
+
+session = DashSession()
```

Comment:
In general, it is considered good practice to always end files in a newline. This is not a python thing, but for all text files. :smile:

VSCode can do it automatically, I think if format on save is enabled.